sites/website/docs/api/fast-components.fastaccordion.md

```diff
@@ -0,0 +1,16 @@
+---
+id: fast-components.fastaccordion
+title: FASTAccordion class
+hide_title: true
+---
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[@microsoft/fast-components](./fast-components.md) &gt; [FASTAccordion](./fast-components.fastaccordion.md)
+
+## FASTAccordion class
```

Comment:
I'm a little concerned here that using "FASTAccordion" is hard to read at a glance.  Could we use break the words apart "FAST Accordion"?  On that note, is it really even necessary to use FAST everywhere?  Clearly that would be a significant change, though feels redundant.

Reply:
This is unrelated to this PR and will be removed when I force-push.